### PR TITLE
Fast add user task

### PR DESCRIPTION
**Pull request by boutell**
cc @paulisloud this should do the trick for you, but I don't have write access in your environment this morning, so please apply this change yourself and let me know.

### CODE REVIEW

**Comment by paulisloud** [comment]
@boutell  Works beautifully! Thanks.

File: lib/modules/apostrophe-pages/public/js/reorganize.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ apos.define('apostrophe-pages-reorganize', {
       // add page depth as data attr
       $li.attr('data-apos-reorganize-depth', node.getLevel());
       if (node.getLevel() !== 1) {
-        $li.find('.apos-reorganize-title').css('padding-left', 15 * node.getLevel() + 'px');
+        $li.find('.apos-reorganize-title').css('padding-left', 30 * node.getLevel() + 'px');
```
Comment (abea) [inline lib/modules/apostrophe-pages/public/js/reorganize.js]: This looks unrelated. What bringing this in intentional?

**Comment by boutell** [comment]
Fixed that issue, merging.
